geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ConvertCommand.scala

```diff
@@ -52,13 +53,13 @@ class ConvertCommand extends Command with MethodProfiling with LazyLogging {
       throw new ParameterException("Missing option: <files>... is required")
     }
 
-    val inputs = params.files.asScala
-    val format = IngestCommand.getDataFormat(params, inputs.toSeq)
+    val inputs = Inputs(params.files.asScala.toSeq)
+    val format = IngestCommand.getDataFormat(params, inputs.paths)
 
     // use .get to re-throw the exception if we fail
-    IngestCommand.getSftAndConverter(params, inputs.toSeq, format, None).get.flatMap { case (sft: SimpleFeatureType, config: com.typesafe.config.Config) =>
-      val files = if (inputs.isEmpty) { StdInHandle.available().iterator } else {
-        inputs.iterator.flatMap(PathUtils.interpretPath)
+    IngestCommand.getSftAndConverter(params, inputs, format, None).get.flatMap { case (sft: SimpleFeatureType, config: com.typesafe.config.Config) =>
+      val files = if (inputs.paths.isEmpty) { StdInHandle.available().iterator } else {
+        inputs.paths.iterator.flatMap(PathUtils.interpretPath)
       }
```

Comment:
I think this is duplicating logic in `Inputs` now, and can just be `inputs.handles.iterator`
```suggestion
      val files = inputs.handles.iterator
```